packages/conventional-changelog/index.js

```diff
@@ -6,7 +6,16 @@ function conventionalChangelog(options, context, gitRawCommitsOpts, parserOpts,
 
   if (options.preset) {
     try {
-      options.config = require('conventional-changelog-' + options.preset.toLowerCase());
+      var scope = '';
+      var name = options.preset.toLowerCase();
+
+      if (name[0] === '@') {
```

Comment:
I'm pretty sure we can use an npm package to achieve this.

Reply:
There is [`is-scoped`](https://www.npmjs.com/package/is-scoped), but it wont separate the package name into it's parts.

Reply:
There is [this packages](https://github.com/keithamus/parse-packagejson-name). Seems like overkill to me though

Reply:
Nor does that package look supported.

Personally, except for `is-scoped`, which looks supported, I wouldn't go with a package that isn't receiving at least some attention once in a while

`is-scoped` probably isn't worth it here either since we already have to do package name manipulation. We don't really help ourselves avoid any complexity.